internal/pkg/levee/levee.go

```diff
@@ -64,7 +70,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 				case callee != nil && conf.IsSink(utils.DecomposeFunction(v.Call.StaticCallee())):
 					for _, s := range sources {
 						if s.HasPathTo(instr.(ssa.Node)) && !s.IsSanitizedAt(v) {
-							report(pass, s, v)
+							items = append(items, reportItems{s, v})
```

Comment:
what does reportItems{s, v} do? I have never seen that syntax before.

Reply:
It's an unkeyed struct literal.

Reply:
Oops, somehow I got confused that reportItems is a slice. 🤦

Reply:
That's happened to me too.

To be fair, I think the literal should be keyed, it would make this snippet more readable.